src/app.tsx

```diff
@@ -295,29 +296,37 @@ export default function Chat() {
 
         {/* Input Area */}
         <form
-          onSubmit={(e) =>
+          onSubmit={(e) => {
+            e.preventDefault();
             handleAgentSubmit(e, {
               data: {
                 annotations: {
                   hello: "world",
                 },
               },
-            })
-          }
-          className="p-3 bg-input-background absolute bottom-0 left-0 right-0 z-10 border-t border-neutral-300 dark:border-neutral-800"
+            });
+            setTextareaHeight("auto"); // Reset height after submission
+          }}
+          className="p-3 bg-gray-50 absolute bottom-0 left-0 right-0 z-10 border-t border-neutral-300 dark:border-neutral-800 dark:bg-neutral-900"
```

Comment:
nit: We use the neutral colors of tailwind so would ideally recommend `bg-neutral-50` instead of `bg-gray-50` but that's just the smallest of nits.